packages/cli/src/constructs/url-assertion.ts

```diff
@@ -3,12 +3,65 @@ import {
   NumericAssertionBuilder,
 } from './internal/assertion'
 
+/**
+ * Sources that can be used for URL monitor assertions.
+ * URL monitors only support status code validation.
+ */
 type UrlAssertionSource =
   | 'STATUS_CODE'
 
+/**
+ * Assertion configuration for URL monitors.
+ * Allows validation of HTTP response status codes.
+ */
 export type UrlAssertion = CoreAssertion<UrlAssertionSource>
 
+/**
+ * Builder class for creating URL monitor assertions.
+ * Provides methods to create assertions for validating HTTP responses.
+ * 
+ * URL monitors have limited assertion capabilities compared to API checks.
+ * They can only validate status codes. For more advanced assertions
+ * (headers, body content, response times), use ApiCheck instead.
+ * 
+ * @example
+ * ```typescript
+ * // Check for specific status code
+ * UrlAssertionBuilder.statusCode().equals(200)
+ * 
+ * // Check for successful response (2xx)
+ * UrlAssertionBuilder.statusCode().greaterThan(199)
+ * UrlAssertionBuilder.statusCode().lessThan(300)
+ * 
+ * // Check for any non-error response
+ * UrlAssertionBuilder.statusCode().lessThan(400)
+ * 
+ * // Check for specific client error
+ * UrlAssertionBuilder.statusCode().equals(404)
+ * 
+ * // Check for server errors
+ * UrlAssertionBuilder.statusCode().greaterThan(499)
+ * UrlAssertionBuilder.statusCode().lessThan(600)
+ * ```
+ * 
+ * @see {@link https://www.checklyhq.com/docs/monitoring/http-checks/#assertions | HTTP Check Assertions}
```

Comment:
Returns 404 and I think the URL will be wrong even after docs are released. Checking